pytest_splunk_addon/splunk.py

```diff
@@ -540,52 +619,234 @@ def uf_external(request):
+                str(os.environ.get("PYTEST_XDIST_WORKER"))
+            )
+        )
+        LOGGER.info(
+            "TEST_RUNNER_DIRECTORY : {}".format(
+                str(os.environ.get("TEST_RUNNER_DIRECTORY"))
+            )
+        )
+        current_path = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            "k8s_manifests",
+            "splunk_standalone",
+        )
+        pattern = r"https:\/\/github\.com\/splunk\/splunk-configurations-base-indexes\/releases\/download\/v[0-9.]+\/splunk_configurations_base_indexes-[0-9.]+.spl"
+        try:
+            response = requests.get(
+                "https://api.github.com/repos/splunk/splunk-configurations-base-indexes/releases/latest"
+            )
+            regex_match = re.search(pattern, str(response.json()))
+            if regex_match:
+                os.environ["SC4S_INDEX_URL"] = regex_match.group()
+        except Exception as e:
+            LOGGER.error(
+                "Exception occurred while fetching SC4S_INDEX_URL : {0}".format(e)
+            )
+        print(os.getenv("SC4S_INDEX_URL"))
+        update_k8s_manifest_files(folder=current_path, file="namespace")
+        update_k8s_manifest_files(folder=current_path, file="splunk_standalone")
+        LOGGER.info("Setting up Splunk")
+        kubernetes_helper_splunk = KubernetesHelper()
+        LOGGER.info(current_path)
+        LOGGER.info(current_path + "/namespace_updated.yaml")
+        kubernetes_helper_splunk.create_namespace(
+            file=os.path.join(current_path, "namespace_updated.yaml"),
+            namespace_name=os.getenv("NAMESPACE_NAME"),
+        )
+        kubernetes_helper_splunk.create_k8s_resource_from_yaml(
+            file=os.path.join(current_path, "nginx.yaml"),
+            namespace_name=os.getenv("NAMESPACE_NAME"),
+        )
+        kubernetes_helper_splunk.create_k8s_resource_from_yaml(
+            file=os.path.join(current_path, "nginx_service.yaml"),
+            namespace_name=os.getenv("NAMESPACE_NAME"),
+        )
+        kubernetes_helper_splunk.wait_for_pod_to_get_ready(
+            pod_name="nginx", namespace_name=os.getenv("NAMESPACE_NAME")
+        )
+        destination_location = "/usr/share/nginx/html/"
+        source_location = str(
+            os.path.join(
+                os.getenv("TEST_RUNNER_DIRECTORY"),
+                "tests",
+                "src",
+                str(os.getenv("SPLUNK_ADDON")),
+            )
+        )
+        LOGGER.info(source_location)
+        kubernetes_helper_splunk.copy_files_to_pod(
+            "nginx", os.getenv("NAMESPACE_NAME"), destination_location, source_location
+        )
+        file = os.path.join(current_path, "splunk_standalone_updated.yaml")
+        if ":\\" in source_location:
+            source_location = source_location.split(":\\", 1)[1]
+        if "\\" in source_location:
+            source_location = source_location.replace("\\", "/")
```

Comment:
This one is for windows. So, during addon installation it will try to replicate the same path of the local system where `addon_spl` exists on the nginx pod from which when the splunk will be spinned the addon will get installed.